ArduSub/ArduSub.cpp

```diff
@@ -36,6 +36,7 @@ const AP_Scheduler::Task Sub::scheduler_tasks[] = {
     SCHED_TASK(three_hz_loop,          3,     75),
     SCHED_TASK(update_turn_counter,   10,     50),
     SCHED_TASK(compass_accumulate,   100,    100),
+    SCHED_TASK(update_beacon,         50,     50),
```

Comment:
It is better to make Hertz according to the beacon device.
Is it a sensible value that it matches Rober 's Hertz?